tests/e2e/features/transactions/withdraw.feature

```diff
@@ -91,3 +91,21 @@ Feature: Withdraw
     Then Element with "text" " Learn more " should be "clickable"
     Then Element with "text" " Make another transaction " should be "visible"
     Then Element with "text" " Explore ecosystem " should be "visible"
+    #id1694
```

Comment:
Should we add this ID to the header of the test?

Reply:
Added